internal/server/router/router.go

```diff
@@ -18,16 +18,22 @@ import (
 
 func init() {
 	// See https://github.com/getkin/kin-openapi/issues/640
-	openapi3filter.RegisterBodyDecoder("application/cloudevents+json", jsonBodyDecoder)
+  openapi3filter.RegisterBodyDecoder("application/cloudevents-batch+json", jsonBodyDecoder)
```

Comment:
should we register both openapi3filter.RegisterBodyDecoder("application/cloudevents+json", jsonBodyDecoder) and openapi3filter.RegisterBodyDecoder("application/cloudevents-batch+json", jsonBodyDecoder) in init?